spin-observatory-plugin-deck/src/components/PluginContainer.tsx

```diff
@@ -29,8 +30,7 @@ export function PluginContainer({ app }: IPluginContainerProps) {
   };
 
   const handleDateFilterChange = ({ start, end }: { start: number, end: number }) => {
-    // eslint-disable-next-line no-console
-    console.log("filter executions ", { start, end });
+    setSelectedDateFilter({ start: start, end: end });
```

Comment:
JS has a nice shortcut where you can just say `{ start, end }` instead of `{ start: start, end: end }` if both the object key and the variable name of the value are the same like they are here